src/MLDeployment.php

```diff
@@ -27,6 +29,7 @@ public static function fromArray(array $in): self
             ->setModelVersion($in['modelVersion'] ?? '')
             ->setModelStage($in['modelStage'] ?? '')
             ->setUrl($in['url'] ?? '')
+            ->setTrackingTokenId($in['trackingTokenId'] ?? '')
```

Comment:
If you create object using `MLDeployment::fromArray([])`, `toArray` will return `{trackingTokenId: ""}` and causes the token to reset. I think `$in['trackingTokenId'] ?? null` would be better.

Reply:
It would also deserve a unit test

Reply:
ok, thanks